### PR TITLE
New multi-arch WildFly S2I images

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

**Comment by jfdenise** [comment]
@bstansberry @jmesnil @jamezp blog post for multi-arch images. Thank-you.

**Comment by darranl** [comment]
We have a couple of blog posts describing how naming conventions have changed - could these include a link to the instructions describing how to use these images in case these posts become the first place users find?

**Comment by jmesnil** [comment]
+1, for the docker images, the container registry explains the naming convention: https://quay.io/repository/wildfly/wildfly?tab=info

We could do the same for the S2I builder and runtime images

File: _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+author: jfdenise
+---
+
+This article provides details on the new S2I and runtime multi-arch images.
+
+== New WildFly S2I and Runtime Multi-arch images
+
+These new multi-arch images (`linux/arm64` in addition to `linux/amd64`) have a different naming scheme than the current WildFly images 
+to better handle multiple JDK versions and align with the tag scheme used for the WildFly centos7 docker images 
+(as explained in https://www.wildfly.org/news/2022/11/10/wildfly-docker-temurin/[this] blog post).
```
Comment (bstansberry) [inline _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc:16]: This is just a personal opinion, others might disagree, but I found using the title of the post "Updates on WildFly Docker Images" more comfortable to read than 'this'. Perhaps even personalize with 'as explained in Jeff Mesnil's https://....[Updates on WildFly Docker Images] blog post.'

File: _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+NOTE: The previous WildFly images are now deprecated and are no more updated.
```
Comment (bstansberry) [inline _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc:18]: s/no more/no longer/g

File: _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+== Deprecated snapshot multi-arch images
+
+Up to now, we were releasing multi-arch images as preview ones in the `quay.io/wildfly-snapshots` organization:
+
+* `quay.io/wildfly-snapshots/wildfly-runtime-jdk11-multi-arch:latest`
+* `quay.io/wildfly-snapshots/wildfly-runtime-jdk17-multi-arch:latest`
+* `quay.io/wildfly-snapshots/wildfly-s2i-jdk11-multi-arch:latest`
+* `quay.io/wildfly-snapshots/wildfly-s2i-jdk17-multi-arch:latest`
+
+These images are now deprecated and no more updated.
```
Comment (bstansberry) [inline _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc:66]: s/no more/no longer/g

File: _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+== Deprecated wildfly images
```
Comment (bstansberry) [inline _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc:68]: s/wildfly/WildFly

File: _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+The following single-arch images are now deprecated and are no more updated:
```
Comment (bstansberry) [inline _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc:70]: s/no more/no longer/g

File: _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+* `quay.io/wildfly/wildfly-runtime-jdk11:<tag>`
+* `quay.io/wildfly/wildfly-runtime-jdk17:<tag>`
+* `quay.io/wildfly/wildfly-s2i-jdk11:<tag>`
+* `quay.io/wildfly/wildfly-s2i-jdk17:<tag>`
+
+== New OpenShift Image streams
```
Comment (bstansberry) [inline _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc:77]: s/streams/Streams

File: _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+link:https://github.com/wildfly/wildfly-s2i/blob/main/imagestreams/wildfly-runtime-jdk11.yaml[wildfly-s2i-jdk17]) have been deprecated.
+
+The new image streams are:
+
+** WildFly s2i builder: link:https://raw.githubusercontent.com/wildfly/wildfly-s2i/main/imagestreams/wildfly-s2i.yaml[wildfl-s2i imagestream]
+** WildFly runtime: link:https://raw.githubusercontent.com/wildfly/wildfly-s2i/main/imagestreams/wildfly-runtime.yaml[wildfly-runtime imagestream]
+
+== Impact on Helm Charts for WildFly
+
+A new release of link:https://github.com/wildfly/wildfly-charts[Helm Chart for WildFly] will be released soon to use the new images.
```
Comment (bstansberry) [inline _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc:91]: s/of/of the/g

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+You can now identify the latest image supporting the latest LTS JDK version (JDK 17 at the time of this writing):
+
+* `quay.io/wildfly/wildfly-runtime:latest`
+* `quay.io/wildfly/wildfly-s2i:latest`
+
+NOTE: Relying on this image tag implies that the JDK version will get automatically updated when a new LTS JDK is 
+released and supported by the WildFly images.
+
+== Deprecated snapshot multi-arch images
```
Comment (bstansberry) [inline _posts/2022-12-05-wildfly-s2i-images-rename-multi-archs.adoc:57]: Snapshot
Multi-arch
Images

**Comment by jfdenise** [comment]
@jmesnil @bstansberry @darranl , thank-you I have incorporated your feedback. I took the opportunity of the new Helm Charts release to advertise it.

**Pull request merged**